Get-help.md

```diff
@@ -48,9 +40,41 @@ If you have a question about a pull request or issue, you can also reach out to
 
 You can even mention whole teams of people! For example, if you find an issue that is destabilizing the project, you could notify all the core maintainers by including `@oppia/core-maintainers` in your issue. The teams are all listed [here](https://github.com/orgs/oppia/teams).
 
+### Email
+
+We have several mailing lists in the form of Google Groups that you can join(not much recommended):
+
+* [oppia-announce](https://groups.google.com/forum/#!forum/oppia-announce) is for announcements of new releases or blog posts. It's not for asking questions though.
+* [oppia-dev](https://groups.google.com/forum/#!forum/oppia-dev) is the main mailing list for communication between developers and for technical questions. You can post to it even if you're not a member of the group. This is where you can ask questions, solicit feedback, or make developer-specific announcements (e.g. a temporary GitHub outage).
+
+You can also email your mentor with any questions. If you don't have a mentor, complete the steps on the [[wiki page for contributing code to Oppia|Contributing-code-to-Oppia]] and you'll be assigned one.
+
 ## How to Ask Good Questions
 
-At Oppia we don’t care how silly your question is! Just ensure your question is clear, and provide us with enough information to help us resolve it faster. We've divided the questions into 2 categories - Setup-related and General questions. You can start following the sections below to understand how you can ask each of them.
+At Oppia we don’t care how silly your question is! Just ensure your question is clear, and provide us with enough information to help us resolve it faster. 
+
+### General Guidelines to follow 
+
+* Be clear and concise: Clearly articulate your question to avoid confusion and allow others to understand it easily.
+* Provide context: If applicable, provide relevant background information to help others understand your question and situation.
+* Be respectful: Maintain a respectful and courteous tone in your communication to foster a positive and inclusive community environment.
+* Use proper formatting: Organize your question using paragraphs, bullet points, and appropriate formatting to enhance readability.
+* Proofread your message: Take a moment to proofread your question to ensure clarity and accuracy before sending it.
+
+### Providing Error Information 
+
+When reporting an error, follow these steps to communicate the issue effectively:
+
+* Start with the Command: Rather than saying "I have an error," provide the actual command you ran. Copy and paste the command in your email or chat message to give recipients the complete context.
```

Comment:
For the first part, use sentence case -- only capitalize the first letter of the whole phrase. Ditto below.